migrations/20180727173834-add-status-to-order.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+import status from '../server/constants/order_status';
```

Comment:
I agree. I used this convention to match the `expense_status.js` file. I'll create an issue to standardize this.

Reply:
After having looked at several repositories, looks like the current common practice is ... hyphen case! So that would be `order-status`. Anyway, let's take care of that later.